hyperspy/io_plugins/edax.py

```diff
@@ -0,0 +1,935 @@
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+# The details of the format were taken from
+# http://www.biochem.mpg.de/doc_tom/TOM_Release_2008/IOfun/tom_mrcread.html
+# and http://ami.scripps.edu/software/mrctools/mrc_specification.php
+
+import os
+import numpy as np
+from hyperspy.misc.array_tools import sarray2dict
+import traits.api as t
+
+# Plugin characteristics
+# ----------------------
+format_name = 'EDAX TEAM'
+description = 'Reader for EDS maps and spectra saved by the EDAX TEAM' \
+              'software: \n' \
+              'An SPD file contains map data. The spectral information is ' \
+              '\nheld in an SPC file with the same name, while the spatial ' \
+              '\ncalibration is held in a related IPR file. If an SPD file ' \
+              '\nis loaded, the result will be a Hyperspy EDSSpectrum map, ' \
+              '\nand the calibration will be loaded from appropriate SPC ' \
+              '\nand IPR files (if available). \n' \
+              'If an SPC file is loaded, the result will be a single \n' \
+              'EDSSpectrum with no other files needed for calibration.'
+full_support = False
+# Recognised file extension
+file_extensions = ['spd', 'SPD', 'spc', 'SPC']
+default_extension = 0
+
+# Writing capabilities
+writes = False
+
+spd_extensions = ('spd', 'SPD')
+spc_extensions = ('spc', 'SPC')
+
+atomic_num_dict = {1: 'H', 2: 'He', 3: 'Li', 4: 'Be', 5: 'B', 6: 'C', 7: 'N',
+                   8: 'O', 9: 'F', 10: 'Ne', 11: 'Na', 12: 'Mg', 13: 'Al',
+                   14: 'Si', 15: 'P', 16: 'S', 17: 'Cl', 18: 'Ar', 19: 'K',
+                   20: 'Ca', 21: 'Sc', 22: 'Ti', 23: 'V', 24: 'Cr', 25: 'Mn',
+                   26: 'Fe', 27: 'Co', 28: 'Ni', 29: 'Cu', 30: 'Zn', 31: 'Ga',
+                   32: 'Ge', 33: 'As', 34: 'Se', 35: 'Br', 36: 'Kr', 37: 'Rb',
+                   38: 'Sr', 39: 'Y', 40: 'Zr', 41: 'Nb', 42: 'Mo', 43: 'Tc',
+                   44: 'Ru', 45: 'Rh', 46: 'Pd', 47: 'Ag', 48: 'Cd', 49: 'In',
+                   50: 'Sn', 51: 'Sb', 52: 'Te', 53: 'I', 54: 'Xe', 55: 'Cs',
+                   56: 'Ba', 57: 'La', 58: 'Ce', 59: 'Pr', 60: 'Nd', 61: 'Pm',
+                   62: 'Sm', 63: 'Eu', 64: 'Gd', 65: 'Tb', 66: 'Dy', 67: 'Ho',
+                   68: 'Er', 69: 'Tm', 70: 'Yb', 71: 'Lu', 72: 'Hf', 73: 'Ta',
+                   74: 'W', 75: 'Re', 76: 'Os', 77: 'Ir', 78: 'Pt', 79: 'Au',
+                   80: 'Hg', 81: 'Tl', 82: 'Pb', 83: 'Bi', 84: 'Po', 85: 'At',
+                   86: 'Rn', 87: 'Fr', 88: 'Ra', 89: 'Ac', 90: 'Th', 91: 'Pa',
+                   92: 'U', 93: 'Np', 94: 'Pu', 95: 'Am', 96: 'Cm', 97: 'Bk',
+                   98: 'Cf', 99: 'Es', 100: 'Fm', 101: 'Md', 102: 'No',
+                   103: 'Lr', 104: 'Rf', 105: 'Db', 106: 'Sg', 107: 'Bh',
+                   108: 'Hs', 109: 'Mt', 110: 'Ds', 111: 'Rg', 112: 'Cp',
+                   113: 'Uut', 114: 'Uuq', 115: 'Uup', 116: 'Uuh', 117: 'Uus',
+                   118: 'Uuo'}
```

Comment:
Could be replaced by:
```python
from hyperspy.misc.elements import elements_db
atomic_num_dict = dict(((p.General_properties.Z, e) for (e, p) in elements_db)
```
But completely unimportant :)
(also the database stops at U, but that can be extended if need be)